### PR TITLE
Update various docs for 2nd line

### PR DESCRIPTION
**Pull request by deborahchua**
https://trello.com/c/kZj6C6Pa/41-set-up-wiki-page-for-internal-dev-docs
https://trello.com/c/2Kzg5T40/1036-incorrect-postcode-data-responding-to-zendesk-tickets
https://trello.com/c/JTFHqUKA/1172-add-to-docs-log-retention-period-for-sentry-and-kibana

### CODE REVIEW

File: source/manual/incorrect-postcode-data.html.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Incorrect postcode data
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+---
+
+We use postcode data from the ONS Postcode Directory (https://geoportal.statistics.gov.uk/search?q=ONS%20Postcode%20Directory%20(ONSPD) and Ordnance Survey for boundary
```
Comment (Rosa-Fox) [inline source/manual/incorrect-postcode-data.html.md:9]:
```suggestion
We use postcode data from the [ONS Postcode Directory] (https://geoportal.statistics.gov.uk/search?q=ONS%20Postcode%20Directory%20) (ONSPD) and Ordnance Survey for boundary
```

File: source/manual/incorrect-postcode-data.html.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+line data. We typically [import the data into Mapit](https://github.com/alphagov/mapit/blob/master/IMPORTING-DATA.md) every 6 months.
+
+We sometimes receive Zendesk tickets about postcodes not returning the right results.
+You can check the accuracy of the postcode by:
+
+* Looking up the postcode on Mapit https://mapit.mysociety.org/postcode/<postcode>
```
Comment (Rosa-Fox) [inline source/manual/incorrect-postcode-data.html.md:15]: For some reason the `<postcode>` part of the URL disappears when the markdown render.

mapit-url

Maybe change to: 

```
Looking up the postcode on Mapit, for example https://mapit.mysociety.org/postcode/E18QS and having a look at the parents.
```

File: source/manual/sentry.html.md [hunk 3]
```diff
@@ -292,3 +292,7 @@ Sentry is used by several programmes in GDS, not just GOV.UK. A report,
 [GDS use of Sentry.io](https://docs.google.com/document/d/1yVa9iiu-DayGy-MtlrdXeV0E09CsetUenZJCmusLfLg/edit),
 covers this in more detail, including documenting some of the limitations of the
 setup.
+
+## Retention period
+
+[90 days](https://sentry.io/security/#data-retention).
```
Comment (Rosa-Fox) [inline source/manual/sentry.html.md:298]: Do you know where this is defined? Is it in the Sentry UI or in code somewhere? I had a quick look in govuk-apppconfig but couldn't see it. Could be useful to link to that and then a separate link to the Sentry data retention policy?

**Review by Rosa-Fox — COMMENTED** [review]
Nice one @deborahchua :) I had a couple of small comments. Also I realised my Atlasssian access is revoked! So I couldn't check some of the links but guessing they are fine.

File: source/manual/sentry.html.md (hunk 3 above)
Comment (deborahchua) [reply source/manual/sentry.html.md:298]: Yeah I also didn't have luck finding it either(!) so I've just linked to the standard Sentry policy for now.

Comment (Rosa-Fox) [reply source/manual/sentry.html.md:298]: 👍

**Review by Rosa-Fox — APPROVED** [review]

**Pull request merged**